packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -77,13 +77,13 @@
 }
 
 .mdc-chip__icon--trailing {
-  margin: 0 -4px 0 4px;
+  margin: $mdc-chip-trailing-icon-margin-top $mdc-chip-trailing-icon-margin-right $mdc-chip-trailing-icon-margin-bottom $mdc-chip-trailing-icon-margin-left;
```

Comment:
this is basically duplicating the logic in `mdc-chip-trailing-icon-margin`...so instead call `mdc-chip-trailing-icon-margin` from `@at-root`

and then re-run the screenshots :)